tools/federation/src/test/java/org/eclipse/rdf4j/federated/LimitTests.java

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Eclipse RDF4J contributors.
```

Comment:
Could you update the year to 2024?

Reply:
ah, again copy & paste from a parallel file. Will do immediately. Good spot.

Have squashed it into the existing commit